tests/api2/test_007_pool.py

```diff
@@ -192,13 +232,15 @@ def test_12_import_nonascii_msdosfs():
     expect_state(job_id, "SUCCESS")
 
 
-def test_13_look_if_Каталог_slash_Файл():
+def test_15_look_if_Каталог_slash_Файл(request):
+    depends(request, ["pool_04"])
     cmd = f'test -f {dataset_path}/Каталог/Файл'
     results = SSH_TEST(cmd, user, password, ip)
     assert results['result'] is True, results['output']
 
 
-def test_14_import_ntfs():
+def test_16_import_ntfs(request):
```

Comment:
Need to skip this on TrueNAS Enterprise or "HA" based systems as it's a non-supported operation.

Reply:
I move it out of the pool test and move it in pool_import and mark it to skip all pool import test on HA.